packages/integrations/tailwind/README.md

```diff
@@ -61,6 +61,52 @@ export default defineConfig({
 });
 ```
 
+Then, create a `tailwind.config.cjs` file in your project's root directory. You can use the following command to generate a basic configuration file for you:
+
+```sh
+npx tailwindcss init
+```
+
+Add this basic configuration to your `tailwind.config.cjs` file:
+
+```js ins={4} "content: ['./src/**/*.{astro,js,ts,tsx,md,mdx}']"
+// tailwind.config.cjs
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
+```
+
+Finally, import your `base.css` file inside your Layout component:
+
+```astro ins={2} "import '../styles/global.css';"
+---
+import "../styles/base.css";
+interface Props {
+  title: string;
+}
+const { title } = Astro.props;
+---
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="description" content="Astro description" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <meta name="generator" content={Astro.generator} />
+    <title>{title}</title>
+  </head>
+  <body>
+    <slot />
+  </body>
+</html>
+```
```

Comment:
I think we can remove this too as it's covered by `applyBaseStyles`?

Reply:
woops, my bad.